openfl-workspace/torch_cnn_mnist/README.md

```diff
@@ -121,6 +120,43 @@ class MNISTDataset(ImageFolder):
         else:
             return super(MNISTDataset, self).__getitem__(index)
 ```
+Use a pre-sharded version of the standard MNIST dataset (divided among two data owners). The pre-sharded dataset can be downloaded from [mnist_data_shards.tar.gz](https://github.com/securefederatedai/openfl-contrib/blob/main/openfl_contrib_tutorials/ml_to_fl/federated/mnist_data_shards.tar.gz). 
+
+Copy the dataset bundle to the root of the FL workspace and unpack it:
+```
+cp mnist_data_shards.tar.gz ~/openfl/openfl-workspace/torch_cnn_mnist/
+cd ~/openfl/openfl-workspace/torch_cnn_mnist/
```

Comment:
Remember that, as per the first paragraph, the workspace is instantiated in the `~/openfl-quickstart/fl_workspace` folder. Please adjust it here accordingly,

Reply:
fixed in c9bb19088383b978a147804539dee0d7dd37b515